minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/MissionHandlers/CameraCommandsImplementation.java

```diff
@@ -46,28 +48,34 @@ public void setOverriding(boolean b) {
 
     @Override
     protected boolean onExecute(String verb, String parameter, MissionInit missionInit) {
-        if (!verb.equals("camera"))
-            return false;
         try {
-            String[] camParams = parameter.split(" ");
-
-            float pitch = Float.parseFloat(camParams[0]);
-            float yaw = Float.parseFloat(camParams[1]);
-            EntityPlayerSP player = Minecraft.getMinecraft().player;
-            if (player != null) {
-                this.currentYaw = player.rotationYaw;
-                this.currentPitch = player.rotationPitch;
+            if (verb.equals("camera")) {
+                String[] camParams = parameter.split(" ");
+                double pitch = Double.parseDouble(camParams[0]);
+                double yaw = Double.parseDouble(camParams[1]);
+                if (MalmoMod.isLowLevelInput()) {
+                    double sensitivity = 3.0;
+                    if (yaw != 0.0 || pitch != 0.0) {
+                        FakeMouse.addMovement((int) (sensitivity * yaw), -(int) (sensitivity * pitch));
+                    }
+                } else {
+                    EntityPlayerSP player = Minecraft.getMinecraft().player;
+                    if (player != null) {
+                        this.currentYaw = player.rotationYaw;
+                        this.currentPitch = player.rotationPitch;
 
-                player.setPositionAndRotation(player.posX, player.posY, player.posZ, this.currentYaw + yaw, this.currentPitch + pitch);
+                        player.setPositionAndRotation(player.posX, player.posY, player.posZ, (float) (this.currentYaw + yaw), (float) (this.currentPitch + pitch));
 
-                this.currentYaw = player.rotationYaw;
-                this.currentPitch = player.rotationPitch;
+                        this.currentYaw = player.rotationYaw;
+                        this.currentPitch = player.rotationPitch;
+                    }
+                }
+                return true;
             }
         } catch (NumberFormatException e) {
             System.out.println("ERROR: Malformed parameter string (" + parameter + ") - " + e.getMessage());
```

Comment:
This is ignoring number format exceptions then? That is expected right?

Reply:
my understanding is that, just like prior to the change, in case of NumberFormatExceptions, we print an error message and return false (which, AFAIK, means that the command will be passed further down the chain)